spec/abstractops.html

```diff
@@ -15,6 +15,7 @@ <h1>GetOption ( _options_, _property_, _values_, _fallback_ )</h1>
     <emu-alg>
       1. If _options_ is *undefined* or *null*, then
         1. Return _fallback_.
+      1. Assert: Type(_options_) is Object.
```

Comment:
Is this the only change you made to the spec?

You should not use `Assert` except for an actual assertion.  You should instead do something like,

```
1. If Type(_options_) is Object, throw a *TypeError* exception.
```

Reply:
That assertion actually already made it to ECMA-402 through <https://github.com/tc39/ecma402/pull/465>.

Even though I don't think that change should have happened in the first place, because adding assertions for just a single parameter seems dubious. Also see the discussion in <https://github.com/tc39/ecma262/pull/2050> about mismatched assertions. 

IOW if assertions are really needed for that operation, they should at least be consistent: [GetOption](https://tc39.es/ecma402/#sec-getoption) calls [Get](https://tc39.es/ecma262/#sec-get-o-p), so normally it's directly clear (*) that `options` need to be an Object and `property` a PropertyKey, cf. the assertions in [Get](https://tc39.es/ecma262/#sec-get-o-p) itself. Only adding an assertion for the `object` parameter is strange, there should at least also be the assertion `Assert: IsPropertyKey(property) is true.`. 

(*) If a non-object parameter was expected here, the operation would have called [GetV](https://tc39.es/ecma262/#sec-getv) instead.

Reply:
In 262, both parameters would have implicit assertions in the AO prose.